lang/src/org/partiql/lang/planner/PlannerPipeline.kt

```diff
@@ -231,18 +276,21 @@ interface PlannerPipeline {
                 )
             }
 
-            val builtinFunctions = createBuiltinFunctions(valueFactory)
-            // TODO: uncomment when DynamicLookupExprFunction exists
-//            val builtinFunctions = createBuiltinFunctions(valueFactory) + DynamicLookupExprFunction()
+            val builtinFunctions = createBuiltinFunctions(valueFactory) + DynamicLookupExprFunction()
             val builtinFunctionsMap = builtinFunctions.associateBy {
                 it.signature.name
             }
 
+            // customFunctions must be on the right side of + here to ensure that they overwrite any
+            // built-in functions with the same name.
```

Comment:
I  know this for later but I'm noting—overwrite any built-int—as a design decision that we need to make on the open type system.